Sources/SwiftUITooltip/config/ArrowOnlyTooltipConfig.swift

```diff
@@ -18,9 +18,13 @@ public struct ArrowOnlyTooltipConfig: TooltipConfig {
     public var height: CGFloat?
 
     public var borderRadius: CGFloat = 8
+    public var borderRadiusStyle: RoundedCornerStyle = .circular
     public var borderWidth: CGFloat = 0
     public var borderColor: Color = Color.primary
     public var backgroundColor: Color = Color.clear
+    public var shadowColor: Color = .clear
+    public var shadowRadius: CGFloat = 0
+    public var shadowOffset: CGPoint = .zero
```

Comment:
Let's set this to something nicer so that when a person sets the color to something visible, it becomes a nice shadow, not just four colored sharp corners. I guess that nice values would be `shadowRadius = 8` and `offset.x = 4; offset.y = 4` ?

Reply:
Thanks for the review!
I'm not sure I exactly understand what you mean. Can you provide an example?

Reply:
It's been a while since I touched SwiftUI, so maybe I forgot how shadowRadius works, but as I understand if you just change the color of the shadow to something that is visible, it's not going to look nice, you'll also need to adjust `shadowRadius` and `offset`? Or will the shadow not appear until you set `shadowRadius` to a non-zero value?

Reply:
The shadow will in fact not appear until shadow radius is non-zero. I've added a preview code for the shadow, you can test it :)
P.S. sorry for the late replies, been quite busy these past few days

Reply:
Understandable, I have the same problem with replying on time 😅 

I think we can merge this PR then.